### PR TITLE
When deleting groups, destroy groups that aren't in use by anyone

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

File: app/forms/destroy_group_assignments.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@ def call
     GroupAssignment
       .where(user_id: user_ids, group_id: group_id)
       .delete_all
+    group = Group.find(group_id)
```
Comment (benlovell) [inline app/forms/destroy_group_assignments.rb]: Can you extract this into a private method like `destroy_orphaned_group(group_id)` perhaps? Also leave a bit of breathing space between the deletion of groups and the call.

**Comment by benbarnett** [comment]
Yep this'll work 👍

File: app/forms/destroy_group_assignments.rb (hunk 1 above)
Comment (AndrewVos) [inline app/forms/destroy_group_assignments.rb]: :checkered_flag:

**Comment by benlovell** [comment]
Noice.

**Pull request merged**